code/datums/diseases/advance/symptoms/necropolis.dm

```diff
@@ -49,19 +50,21 @@
 		if(5)
 			if(tendrils)
 				tendril(A)
-			M.dna.species.punchdamagelow = 5
-			M.dna.species.punchdamagehigh = 15
-			M.dna.species.punchstunthreshold = 11
-			M.dna.species.brutemod = 0.6
-			M.dna.species.burnmod = 0.6
-			M.dna.species.heatmod = 0.6
-			M.add_movespeed_modifier(MOVESPEED_ID_NECRO_VIRUS_SLOWDOWN, update=TRUE, priority=100, multiplicative_slowdown=0.5)
-			ADD_TRAIT(M, TRAIT_PIERCEIMMUNE, DISEASE_TRAIT)
-			if(fireproof)
-				ADD_TRAIT(M, TRAIT_RESISTHEAT, DISEASE_TRAIT)
-				ADD_TRAIT(M, TRAIT_RESISTHIGHPRESSURE, DISEASE_TRAIT)
-				M.weather_immunities |= "ash"
-				M.weather_immunities |= "lava"
+			if(!(fullpower)) 							//if we haven't gotten the buff yet
+				M.dna.species.punchdamagelow += 4
+				M.dna.species.punchdamagehigh += 4
+				M.dna.species.punchstunthreshold += 1 	//Makes standard punches 5-14 with higher stun chance (1-10, stun on 10 -> 5-14, stun on 11-14)
+				M.dna.species.brutemod *= 0.6
+				M.dna.species.burnmod *= 0.6
+				M.dna.species.heatmod *= 0.6
+				M.add_movespeed_modifier(MOVESPEED_ID_NECRO_VIRUS_SLOWDOWN, update=TRUE, priority=100, multiplicative_slowdown=0.5)
+				ADD_TRAIT(M, TRAIT_PIERCEIMMUNE, DISEASE_TRAIT)
+				fullpower = TRUE
```

Comment:
move this to line 54 so we know what's going on

Reply:
done